esp/src/src-react/components/AvailablePermissions.tsx

```diff
@@ -0,0 +1,235 @@
+            autoSave: true,
+            canEdit: function (object, value) { return object.__hpcc_type !== "Permission"; },
+            renderHeaderCell: function (node) {
+                node.innerHTML = nlsHPCC.AllowWrite;
+            }
+        }, CheckBox),
+        allow_full: editor({
+            width: 54,
+            editor: "checkbox",
+            editorArgs: { value: true },
+            className: "hpccCentered",
+            autoSave: true,
+            canEdit: function (object, value) { return object.__hpcc_type !== "Permission"; },
+            renderHeaderCell: function (node) {
+                node.innerHTML = nlsHPCC.AllowFull;
+            }
+        }, CheckBox),
+        padding: {
+            width: 20,
+            label: " "
+        },
+        deny_access: editor({
+            width: 54,
+            editor: "checkbox",
+            editorArgs: { value: true },
+            className: "hpccCentered",
+            autoSave: true,
+            canEdit: function (object, value) {
+                if (object.__hpcc_type === "Permission" || object.AccountName === "Administrators") {
+                    return false;
+                }
+                return true;
+            },
+            renderHeaderCell: function (node) {
+                node.innerHTML = nlsHPCC.DenyAccess;
+            }
+        }, CheckBox),
+        deny_read: editor({
+            width: 54,
+            editor: "checkbox",
+            editorArgs: { value: true },
+            className: "hpccCentered",
+            autoSave: true,
+            canEdit: function (object, value) {
+                if (object.__hpcc_type === "Permission" || object.AccountName === "Administrators") {
+                    return false;
+                }
+                return true;
+            },
+            renderHeaderCell: function (node) {
+                node.innerHTML = nlsHPCC.DenyRead;
+            }
+        }, CheckBox),
+        deny_write: editor({
+            width: 54,
+            editor: "checkbox",
+            editorArgs: { value: true },
+            className: "hpccCentered",
+            autoSave: true,
+            canEdit: function (object, value) {
+                if (object.__hpcc_type === "Permission" || object.AccountName === "Administrators") {
+                    return false;
+                }
+                return true;
+            },
+            renderHeaderCell: function (node) {
+                node.innerHTML = nlsHPCC.DenyWrite;
+            }
+        }, CheckBox),
+        deny_full: editor({
+            width: 54,
+            editor: "checkbox",
+            editorArgs: { value: true },
+            className: "hpccCentered",
+            autoSave: true,
+            canEdit: function (object, value) {
+                if (object.__hpcc_type === "Permission" || object.AccountName === "Administrators") {
+                    return false;
+                }
+                return true;
```

Comment:
Change to
```
return object.__hpcc_type !== "Permission" && object.AccountName !== "Administrators";
```

"Permission" and "Administrators" should be made module const declarations.